### DISCUSSION TITLE
负载均衡

### DISCUSSION THREAD

**Discussion by Passerby1011** [immersive-translate/immersive-translate#448] [Ideas]
### 新功能 | New Feature

1.翻译时对用户选择的接口做负载均衡
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.使用网页翻译的情况下，请求量很大
2.有的接口就是慢
3.有的用多了，直接“请求太多的”
4.还有可能遇到的网络问题

### 提议 | Proposal

根据用户选择的接口做负载均衡，直接对所有要翻译的内容，按选择的接口数量按区域做划分。分配给不同的接口，同时给出优先，网页前面的用户最先浏览的部分的分配给那些翻译快的优先级高的

**Comment by xingty** [comment]
新版本的沉浸式翻译已经可以自定义设置DeeplX接口的QPS，你可以尝试一下使用我写的[transcat](https://github.com/xingty/transcat)，可以实现组合任意翻译服务做负载均衡。
